editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -737,6 +737,8 @@ export const NavigatorItem: React.FunctionComponent<
 
   const isComponentScene = useIsProbablyScene(navigatorEntry) && childComponentCount === 1
   const isRenderProp = isRenderPropNavigatorEntry(navigatorEntry)
+  const isRenderPropSlot =
+    isSyntheticNavigatorEntry(navigatorEntry) && navigatorEntry.renderProp != null
```

Comment:
is the synthetic-ness here the thing that makes this a slot? because if it would be filled, we would have a non-synthetic entry for the render prop?

Reply:
"Synthetic" should have been called "Label" or something similar, it means that it's not a navigator entry for something actually in the project, but just something that was "synthesised". The navigator entry for the (empty) slot is independent from the corresponding synthetic navigator entry (the label)

Reply:
> The navigator entry for the (empty) slot is independent from the corresponding synthetic navigator entry (the label)

but the check is literally isSyntheticNavigatorEntry && navigatorEntry.renderProp, so I'm still confused then.